src/Clustering/View/GMUDefaultClusterRenderer.h

```diff
@@ -57,6 +57,16 @@ NS_ASSUME_NONNULL_BEGIN
  * Default to YES.
  */@property(nonatomic) BOOL animatesClusters;
 
+/**
+ * Allows setting a zIndex value for the clusters.  This becomes useful
+ * when using multiple cluster data sets on the map and require a predictable
+ * way of displaying multiple sets with a predictable layering order.
+ *
+ * If no specific zIndex is not specified during the initialization, the
+ * default zIndex is '1'.  Larger zIndex values are drawn over lower ones
+ * similar to the zIndex value of GMSMarkers.
+ */@property(nonatomic) float zIndex;
```

Comment:
Thanks for making the changes.
The zIndex is of type int so we'd like to make this property an int.